honeycombio/data_source_query.go

```diff
@@ -85,8 +86,29 @@ func dataSourceHoneycombioQuery() *schema.Resource {
 							ValidateFunc: validation.StringInSlice(validQueryFilterOps, false),
 						},
 						"value": {
-							Type:     schema.TypeString,
-							Optional: true,
+							Type:        schema.TypeString,
```

Comment:
Note that having both `value` and `value_string` is redundant.
It would probably make more sense to drop `value`, but that would be a breaking change.

Also `value` will likely be the primary option once the honeycomb API is updated to do type mapping.

Reply:
Agree we best keep `value` around as well.